### DISCUSSION TITLE
Add a Play/Pause toggle BT command

### DISCUSSION THREAD

**Discussion by Guillemete** [pschatzmann/ESP32-A2DP#509] [Q&A] [answered]
Hi, 
i have a working prototype that works great with *most app on my iphone. The PLAY button will only send play (i.e. if its paused , it will play, if its already playing, it will let it play)

The only app that doesn't cooperate is netflix. Wheter i hit PLAY or PAUSE command, it will just toggle between play/pause.

So my idea was to use the state of the media  with the call back, then play if its paused and pause if its playing...
But, this would only work if there is an actual PLAY/PAUSE toggle, since the other apps a behaving as they should with the PLAY and PAUSE command
    
So, is possible to add a PLAY/PAUSE toggle to the list of BT commands?

play();
    pause();
    stop();
    next();
    previous();

**Comment by pschatzmann** [comment, marked as answer]
From my point of view this does not make any sense and you can easily implement this logic just in your sketch with a few lines of code

**Reply by Guillemete** [reply]
Thanks for answering !
I get that a logic with the state on splaying would be the way to go. The only thing, is that the netflix app doesn't behave like the rest

For my project , i need to have separate Play and pause, because, depending of the situation, i could end up sending Pause twice in a row. so the normal behavior would keep the app on pause. But with netflix, both the play and pause command toggle, so the app would star playing again.

The work around, as you suggest would be to add a simple logic in the script. but to be "universal" i would need to have a play/pause toggle, so i could simply use the state of playing call back

**Comment by pschatzmann** [comment]
As I said, there is nothing missing on my side, that would prevent you from doing this yourself in your sketch...

**Reply by Guillemete** [reply]
i get that your library is working and the bt command are working fine. i'm trying to find a workaround because not all the app are behaving the same way. 

Thanks anyway!
